### ISSUE TITLE
Cards reading order config when interactive

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2404]
## Description

https://jira.tid.es/browse/DSNCORE-2269

Related to:
* https://jira.tid.es/browse/OBVIVO-2635
* https://github.com/Telefonica/mistica-design/discussions/2149

Allow cards to read elements one by one when they are interactive. This should be configurable, the defaul behaviour should be concatenate all elements in a single stop.

## Tasks

- [x] Update specs
- [ ] Design documentation
- [ ] Add to mobile library
- [ ] Add to desktop library
- [ ] Add to UI Kit
- [ ] Android ticket
- [ ] iOS ticket
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2356)

**Comment by yceballost** [comment]
https://jira.tid.es/browse/DSNCORE-2269

**Comment by yceballost** [comment]
added here https://github.com/Telefonica/mistica-web/releases/tag/v16.58.0
